llama-index-integrations/multi_modal_llms/llama-index-multi-modal-llms-anthropic/llama_index/multi_modal_llms/anthropic/utils.py

```diff
@@ -29,7 +30,18 @@
 logger = logging.getLogger(__name__)
 
 
-def infer_image_mimetype(image_file_path: str) -> str:
+def infer_image_mimetype_from_base64(base64_string):
+    # Decode the base64 string
+    decoded_data = base64.b64decode(base64_string)
+
+    # Use filetype to guess the MIME type
+    kind = filetype.guess(decoded_data)
+
+    # Return the MIME type if detected, otherwise return None
+    return kind.mime if kind is not None else None
```

Comment:
Will returning None break things?

Reply:
I mean, yes, but it was already broken so this is less-broken. You think it should throw an exception if it can't figure it out?

Reply:
Nah it's fine for now, just checking